lib/report/components/css/main.css

```diff
@@ -0,0 +1,231 @@
+/* Layout */
```

Comment:
The main.css file has been broken into its own file. Likely some of this can be moved into the proper .vue files but I didn't tackle that.